optaplanner-docs/src/main/asciidoc/QuickStart/SpringBoot/SpringBootQuickStart-chapter.adoc

```diff
@@ -989,19 +989,19 @@ public class TimeTableController {
 ----
 ====
 +
-For simplicity's sake, this code handles only one `TimeTable`,
+For simplicity's sake, this code handles only one `TimeTable` instance,
 but it is straightforward to enable multi-tenancy and handle multiple `TimeTable` instances of different high schools in parallel.
 +
-The `getTimeTable()` method returns the latest time table from the database.
+The `getTimeTable()` method returns the latest timetable from the database.
```

Comment:
If timetable is a acceptable word, then TimeTable should be renamed to Timetable, across the entire source code, including the quarkus quickstart too? I can do that, just asking.

Reply:
Even if timetable is decent English, I'd keep the time table to timetable change to a separate PR. It will be big.

Reply:
So two things. First, you do not need to change TimeTable in code. It is acceptable, and I believe preferable, to use camel case for code elements. It makes it clear to the reader that it is a code object. Second, there is both timetable and time table in this chapter. only three instance of time table so went ahead an changed. In fact, those were the only three in the entire OptaPlanner doc. There are two SVG diagrams that have it but I didn't change those. We can do that later.

Reply:
Why not simply standarize on "time table" (=TimeTable) everywhere? Don't mix with "timetable" (=Timetable) anywhere

Reply:
Hey, sorry I didn't see the ball come back over to me! Usually, I wouldn't dig in, but I can't find any reputable sources that spell a school schedule as time table. The IBM SG doesn't list either spelling in the word list but do use it in context (pg 161:
"Align times (for example, in a timetable) on colons."

Merriam Webster uses timetable:
https://www.merriam-webster.com/dictionary/timetable

The only references I find are the less common 'time tables' to refer to multiplication tables (usually 'times tables'). So I think that if we use 'timetable' in the text and use TimeTable in code elements, the distinction is clear. But if you are still uncomfortable, let me run it by Stetson on when he is back at work later in the week.

Reply:
If timetable is a acceptable word, then TimeTable should be renamed to Timetable, across the entire source code, including the quarkus quickstart too. Let's do that in a separate PR? I can handle it with regex's.